cmd/dm-ctl/main.go

```diff
@@ -193,6 +194,10 @@ func interactionMode() {
 		}
 	}()
 
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	go common.SyncMasterEndpoints(ctx)
```

Comment:
How about only doing this for `interactionMode`?

Reply:
It's only for interactionMode.